packages/legend-studio/src/components/editor/edit-panel/mapping-editor/MappingExecutionBuilder.tsx

```diff
@@ -399,12 +415,12 @@ export const MappingExecutionRelationalInputDataBuilder = observer(
     const updateInput = (val: string): void =>
       inputDataState.inputData.setData(val);
 
-    // TODO: handle CSV input type
-
     return (
```

Comment:
Are we allowing the same flow for the execution panel? Users should also be able to change to csv/sql in the execution editor as well

Reply:
Yes , that is what I did, I am not sure though if we would want to do that or not